03/app.js

```diff
@@ -0,0 +1,14 @@
+const btn = document.querySelectorAll('button');
+console.log(btn)
+
+
+const clickBtn = function () {
+    this.innerText = 'clicked';
+    console.log('clicked')
```

Comment:
Tutaj trzeba coś dopisać. Jeśli klikasz ponownie w button to w konsoli znów widać napisz.
Powinien być on widoczny tylko raz. Należy usunąć nasłuchiwanie (event) po pierwszym kliknięciu.